package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "zOS",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

```suggestion
  "version": "0.10.0",
```
We shouldn't need to increase the version number unless we want to trigger a Production release.